aws/resource_aws_batch_job_definition.go

```diff
@@ -56,6 +56,20 @@ func resourceAwsBatchJobDefinition() *schema.Resource {
 					},
 				},
 			},
+			"timeout": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"attempt_duration_seconds": {
+							Type:     schema.TypeInt,
+							Required: true,
```

Comment:
According to the AWS documentation, this is an optional field, see https://docs.aws.amazon.com/batch/latest/APIReference/API_JobTimeout.html

Reply:
Correct, both timeout and attempt_duration_seconds are optional. I guess the idea is that at some point in the future AWS might add more ways to specify a timeout (eg. attempt_duration_minutes), in which case the individual fields can not be required.